packages/forma-36-codemod/utils/changeImportWithComponentName.js

```diff
@@ -0,0 +1,18 @@
+const { addImport } = require('./addImport');
+const { removeComponentImport } = require('./removeComponentImport');
+
+module.exports.changeImportWithComponentName = function changeImport(
+  j,
+  source,
+  { from, to, componentName, outputComponentName },
+) {
+  let result = removeComponentImport(j, source, {
+    componentName,
+    importName: from,
+  });
+
+  const importResult = addImport(j, result, [
+    j.template.statement([`import { ${outputComponentName} } from "${to}"`]),
+  ]);
+  return importResult.source;
```

Comment:
I think this codemod thing is magical, I still need to do one

I have one question about this two functions, I understand that it will replace old imports
but what happens in a scenario like this?

```jsx
import { Button, ComponentToReplace } from '@contentful/f36-components'
```

Will it do this?

```jsx
import { Button, ReplacedComponent } from '@contentful/f36-components'
```

Or will it create a new duplicated import?

```jsx
import { Button } from '@contentful/f36-components'
import { ReplacedComponent } from '@contentful/f36-components'
```

Reply:
@gui-santos I don't understand the scenario tbh. What we do here is to replace:
```
import { Tag } from "@contentful/forma-36-react-components";
```

with 

```
import { Badge } from "@contentful/f36-components";
```

Reply:
it should do `import { Button, ReplacedComponent } from '@contentful/f36-components'`

Reply:
Ok, let's use `Tag` in the example, so what happens when I have something like this?

```jsx
import { Button, Heading, Table, Tag } from '@contentful/f36-components'
```

We only want to replace `Tag`, but it's being imported with other components
And by reading the code I understood (or misunderstood) that we are looking specifically for places in the code where `Tag` is being imported alone

Reply:
This could be covered on the tests, with the fixtures, add another component being imported and checking what the output would be like.